app/Listeners/UserEventsSubscriber.php

```diff
@@ -135,12 +135,14 @@ public function onUpdated(UserUpdated $event): void
         //Update Redisearch websites index
         $this->updateUsersIndex($user);
 
-        logger()->notice('User ' . $user->uuid . ' updated',
-            [
-                'event' => EventType::USER_UPDATED,
-                'user' => $user->uuid,
-            ]
-        );
+        if ($event->sendLog()):
```

Comment:
La versione di if-else con i due punti non è preferibile. In tutta la _codebase_ i blocchi sono racchiusi da parentesi graffe.